DH_Engine/Classes/DHVotingHandler.uc

```diff
@@ -430,7 +433,7 @@ function string SetupGameMap(MapVoteMapList MapInfo, int GameIndex, MapHistoryIn
     // Add Per-GameType Game Options
     if (GameConfig[GameIndex].Options != "")
     {
-        OptionString = OptionString $ Repl(Repl(GameConfig[GameIndex].Options,",","?")," ","");
+        OptionString = OptionString $ Repl(Repl(GameConfig[GameIndex].Options, ", ", "?"), " ", "");
```

Comment:
I think this will actually break functionality? You added a space inside the string where there shouldn't be one.